src/theme/overrides/Button.ts

```diff
@@ -0,0 +1,22 @@
+import { Components, Theme } from "@mui/material";
+
+export const Button = (theme: Theme): Components => {
+  return {
+    MuiButton: {
```

Comment:
Every time you create an overridden component, don't forget about the theme parameter that this function has, if you have the backgroundColor property, it is better to write it like this:
`backgroundColor: theme.palette.primary.main`

Reply:
also before styleOverrides add defaultProps:  { disableRipple: true }